### PR TITLE
MinuteMedia: Add GPP macros

### PR DESCRIPTION
**Pull request by zkosanovic**

### CODE REVIEW

File: static/bidder-info/minutemedia.yaml [hunk 1]
```diff
@@ -14,5 +14,5 @@ capabilities:
       - video
 userSync:
   iframe:
-    url: https://pbs-cs.minutemedia-prebid.com/pbs-iframe?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect={{.RedirectURL}}
+    url: https://pbs-cs.minutemedia-prebid.com/pbs-iframe?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&redirect={{.RedirectURL}}
```
Comment (onkarvhanumante) [inline static/bidder-info/minutemedia.yaml:17]: @zkosanovic  uuid is not appended to redirect url

```
https://pbs-cs.minutemedia-prebid.com/pbs-iframe?gdpr=&gdpr_consent=&us_privacy=&gpp=&gpp_sid=&redirect=https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%3Dminutemedia%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%22%24UID%22
```

Comment (zkosanovic) [reply static/bidder-info/minutemedia.yaml:17]: Hi @onkarvhanumante, can you please elaborate more?

What uuid? What do you expect to happen with it and why?

Comment (onkarvhanumante) [reply static/bidder-info/minutemedia.yaml:17]: > Hi @onkarvhanumante, can you please elaborate more?
> 
> What uuid? What do you expect to happen with it and why?

by `uuid` I meant user id i.e one gets replaced by user macro in redirect url

Comment (zkosanovic) [reply static/bidder-info/minutemedia.yaml:17]: @onkarvhanumante Ah okay. Well, as documented in the line just below the one you commented on, the correct User ID macro is not `"$UID"`, but `[PBS_UID]`.

Your test URL should look like this:

```
https://pbs-cs.minutemedia-prebid.com/pbs-iframe?gdpr=&gdpr_consent=&us_privacy=&gpp=&gpp_sid=&redirect=https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%3Dminutemedia%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%5BPBS_UID%5D
```

**Review by gargcreation1992 — APPROVED** [review]

**Review by onkarvhanumante — APPROVED** [review]

**Pull request merged**